src/core_plugins/kibana/public/discover/index.html

```diff
@@ -60,6 +60,10 @@ <h1 tabindex="0" id="kui_local_breadcrumb" class="kuiLocalBreadcrumb">
 
       <div class="discover-wrapper col-md-10">
         <div class="discover-content">
+          <discover-unsupported-rollup
```

Comment:
and this will use generic `$scope.unsupportedMessage` or similar, instead of referencing rollups directly